usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -57,6 +58,15 @@ def geocode_filter_locations(scope: str, values: list) -> Q:
 def validate_location_keys(values):
     """ Validate that the keys provided are sufficient and match properly. """
     for v in values:
+        state = v.get("state")
+        country = v.get("country")
+        county = v.get("county")
+        district_current = v.get("district_current")
+        district_original = v.get("district_original")
+        if (state is None or country != "USA" or county is not None) and (
+            district_current is not None or district_original is not None
+        ):
+            raise InvalidParameterException(INCOMPATIBLE_DISTRICT_LOCATION_PARAMETERS)
```

Comment:
Same as comment about markdown updates. Probably want to ensure that `district_original` and `district_current` aren't included in the same Location Object